reference/constraints/Valid.rst

```diff
@@ -10,6 +10,7 @@ an object and all sub-objects associated with it.
 +----------------+---------------------------------------------------------------------+
 | Options        | - `traverse`_                                                       |
 |                | - `payload`_                                                        |
+|                | - `groups`_                                                         |
```

Comment:
Wrong order

Reply:
This is tricky ... because the original options are not sorted either ... I'm going to try to fix some of the issues you reported ... but I won't make many changes. I want to submit a PR to 4.2 to change the format of the RST tables (they are a nightmare to maintain right now) and to sort all options (lots of them are not). I cannot do this in 3.4 because it will be impossible to merge in 4.2. Too many conflicts. Thanks for understanding.